shared/actions/chat/thread-content.js

```diff
@@ -386,7 +390,7 @@ function _unboxedToMessage(
       ? _decodeFailureDescription(payload.state.error.typ)
       : null
     // $FlowIssue
-    const messageText: ChatTypes.MessageText = message.outbox.body
+    const messageText = message.outbox.body
```

Comment:
I wonder what the flow issue is...maybe fixing it would have caught the error below! I'll investigate...

Reply:
Yeah this struck me as strange too, it's like `message.lutbox.body` got cast to the wrong type.

Reply:
Oh I see $FlowIssue is above there, maybe someone thought this was a bogus but it was actually giving them a legitmate error.

Reply:
Actually, I just tried your PR, and flow gives me an error that this `$FlowIssue` is unused. So I think you can just remove it! Yeah, sounds like someone previously thought this was a bogus error but it was legit.

Reply:
That's weird, I didn't see any errors when I committed this. I'll remove it though, thanks!

Reply:
I wonder if flow in CI is busted...